onnxruntime/core/platform/posix/env.cc

```diff
@@ -537,8 +594,18 @@ class PosixEnv : public Env {
   }
 
  private:
-  PosixEnv() = default;
+  PosixEnv()  {
+#ifdef CPUINFO_SUPPORTED
+    cpuinfo_available_ = cpuinfo_initialize();
+    if(!cpuinfo_available_) {
+      LOGS_DEFAULT(ERROR) << "cpuinfo_initialize failed to initialize";
```

Comment:
Can the user correct something to fix this error? Should it be INFO instead or WARNING at worst?

Reply:
Will make it INFO